src/mscorlib/src/System/Exception.cs

```diff
@@ -748,7 +656,7 @@ internal void RestoreExceptionDispatchInfo(System.Runtime.ExceptionServices.Exce
 
         private String _className;  //Needed for serialization.  
         private MethodBase _exceptionMethod;  //Needed for serialization.  
-        private String _exceptionMethodString; //Needed for serialization. 
+        private String _exceptionMethodString; //Needed for serialization.
```

Comment:
This field should not be needed with your other changes anymore because of it is unused.